Geospatial.md

```diff
@@ -162,3 +192,18 @@ The axis order of the coordinates in WKB and bounding box stored in Parquet
 follows the de facto standard for axis order in WKB and is therefore always
 (x, y) where x is easting or longitude and y is northing or latitude. This
 ordering explicitly overrides the axis order as specified in the CRS.
+
+# Invalid geospatial values
+
+An invalid geospatial value refers to any of the following cases:
+
+* `null`: A null value in Parquet.
+* A non-`null` value that are encoded in a valid WKB or bounding box format 
```

Comment:
Perhaps it is better to distinguish invalid bbox and invalid WKB value?

Reply:
Agreed. Literally `null` is a valid value from the perspective of Parquet and is ignored by all kinds of min/max stats.